src/app/src/main/resources/metadata/charts/healthcare_demand/layout.jsonata

```diff
@@ -5,29 +5,29 @@
   "shapes": [
     {
       "type": "line",
-      "x0": "2020-02-05T00:00:00.000Z",
-      "y0": params.general_beds,
-      "x1": "2022-12-31T00:00:00.000Z",
-      "y1": params.general_beds,
+      "x0": (data.x)[0],
```

Comment:
Unfortunately, these x0 and y0 values rely on us getting the timeseries values back in date order - which is the case so far! Ideally would do a `$sort` here but that seems to cause a stack overflow...

Reply:
This should work, though not sure how intelligent it is about caching the results:
```suggestion
      "x0": data.$sort(x)[0],
```